bc_obps/registration/api/v1/operators.py

```diff
@@ -57,7 +57,7 @@ def get_operator(request: HttpRequest, operator_id: UUID) -> Tuple[Literal[200],
     description="""Updates the status of a specific operator by its ID.
     The endpoint allows authorized users to update the operator's status and perform additional actions based on the new status.
     If the operator is new and declined, all associated user operators are also declined, and notifications are sent accordingly.""",
-    auth=authorize("authorized_irc_user"),
+    auth=authorize("v1_authorized_edit_irc_user"),
```

Comment:
This is the only update endpoint for `authorized_irc_user` (all the others are get). This endpoint is only used in v1 because in v2 we don't approve/deny the operator, only the admin user.